src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -590,7 +592,7 @@ public void testPlaceInstanceHoldInstanceNotFound(TestContext context) throws Ex
     logger.info("=== Test place instance hold w/ instance not found ===");
 
     Hold hold = PatronMockOkapi.getHold(PatronMockOkapi.instanceId_notFound);
-    int expectedStatusCode = 501;
+    int expectedStatusCode = 404;
```

Comment:
Why has the behaviour of this existing test changed? How is it related to this change, which is to add an extra property to the incoming request?

Reply:
The point here that there was no handling of "post hold for patron on instance level" processing before. I determined that previously server was returning only an error code 501:

  public void placeInstanceHoldHandler(RoutingContext ctx) {
    ctx.response()
      .setStatusCode(501)
      .end();
  }

Whereas the same handler of patron hold creation on items level has some processing with different kind of error responses. I suspect that was uncompleted functionality of route stubbing. That why some of tests from MainVerticleTest contained odd cases like "testPlaceInstanceHoldSuccess" which despite implying success of operation expect a response code 501. I added implementation for instance level handling and as a result changed expected error code for a value returned by the placeItemHoldHandler.